lib/send-html.js

```diff
@@ -0,0 +1,47 @@
+// Copyright IBM Corp. 2016. All Rights Reserved.
+// Node module: strong-error-handler
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+var ejs = require('ejs');
+var read = require('fs').readFileSync;
```

Comment:
We usually don't save `fs` methods to a variable and always spell full `fs.readFileSync`.